conda_libmamba_solver/solver.py

```diff
@@ -219,8 +220,35 @@ def solve_final_state(
                 channels=self._channels,
                 subdirs=self.subdirs,
             )
-        self._setup_solver(index)
 
+        with Spinner(
```

Comment:
Note this is only here to "mock" the metadata loading present in the classic UI. If we put it before it is cleared by libmamba's progress bars. This step takes almost no time but at least we get the `Collect metadata: done` message once finished, which is not too bad :)